simulator/util/dummy_perception_publisher/README.md

```diff
@@ -0,0 +1,43 @@
+# dummy_perception_publisher
+
+## Purpose
+
+This node publishers the result of the dummy detection with the type of perception. if `real_use_param_` is True, the data containing the Feature is delivered, and if False, the data without the Feature is delivered.
```

Comment:
```suggestion
This node publishes the result of the dummy detection with the type of perception. If `real_use_param_` is True, the data containing the Feature is delivered, and if False, the data without the Feature is delivered.
```
Nit

Reply:
fix at 6d8ec832f2bdc0ec61e3b30f0af5e205006e60dc

Reply:
`perception. if` is still lower-case.

Reply:
Sorry, I fix completely in 5ba71acecf69c3781fe9be463c0a7fe2a7c3c88b